viamonvif/device/device_test.go

```diff
@@ -91,3 +92,106 @@ func readBody(r *http.Request) (string, error) {
 	}
 	return buf.String(), nil
 }
+
+func TestTLSVerificationConfig(t *testing.T) {
+	logger := logging.NewTestLogger(t)
+
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		body, err := readBody(r)
+		if err != nil {
+			t.Fatalf("failed to read body: %v", err)
```

Comment:
why not have a test.That here explicitly?

Reply:
Nope changed